README.md

```diff
@@ -59,15 +60,14 @@ The following projects are official OpenJS Foundation projects. If you are inter
 | <img alt="logo of project jQuery Mobile" src="https://github.com/openjs-foundation/artwork/raw/main/projects/jquery_mobile/jquery_mobile-icon-color.svg" width="30" /> | [jQuery Mobile](https://jquerymobile.com/)        |    | [Contributing Guide](https://github.com/jquery/jquery-mobile/blob/HEAD/CONTRIBUTING.md) | [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CODE_OF_CONDUCT.md) |
 | <img alt="logo of project jQuery UI" src="https://github.com/openjs-foundation/artwork/raw/main/projects/jquery_ui/jquery_ui-icon-color.svg" width="30" /> | [jQuery UI](https://jqueryui.com/)                |    | [Contributing Guide](https://github.com/jquery/jquery-ui/blob/HEAD/CONTRIBUTING.md) | [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CODE_OF_CONDUCT.md) |
 | <img alt="logo of project RequireJS" src="https://github.com/openjs-foundation/artwork/raw/main/projects/requirejs/requirejs-logo-stacked-color.svg" width="30" /> | [RequireJS](https://requirejs.org/)               |    | [Contributing Guide](https://requirejs.org/docs/contributing.html) | [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CODE_OF_CONDUCT.md) |
-| <img alt="logo of project Sizzle" src="https://github.com/openjs-foundation/artwork/raw/main/projects/sizzle/sizzle-icon-color.svg" width="30" /> | [Sizzle](https://sizzlejs.com/)                   |    | [Contributing Guide](https://github.com/jquery/sizzle/blob/HEAD/CONTRIBUTING.md) | [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CODE_OF_CONDUCT.md) |
+| <img alt="logo of project Sizzle" src="https://github.com/openjs-foundation/artwork/raw/main/projects/sizzle/sizzle-icon-color.svg" width="30" /> | [Sizzle](https://sizzlejs.com/)                   |    | [Contributing Guide](https://github.com/jquery/sizzle/blob/HEAD/CONTRIBUTING.md) | [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CODE_OF_CONDUCT.md)
```

Comment:
It looks like for some reason (linter?) the last bar (`|`) is removed from this table.